mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -429,6 +429,18 @@ def delete_run(self, run_id):
             run.lifecycle_stage = LifecycleStage.DELETED
             self._save_to_db(objs=run, session=session)
 
+    def _hard_delete_run(self, run_id):
```

Comment:
Can we add documentation here explaining what this method is used for (e.g., it is used to enable the `mlflow gc` garbage collection tool and is not a public API).

Reply:
Done